script.js

```diff
@@ -1,7 +1,56 @@
-console.log("hello script js");
+var inputHappened = function(currentInput){
+var str = currentInput;
+var lastChar = str[str.length -1];
+var temperature = parseInt(str.substring(0,str.length-1), 10);
+if (!isNaN (temperature))
```

Comment:
Nice!
`parseInt` returns either the integer (truthy value) or a NaN (falsy value)
so you can also refactor this to a shorter condition `if (temperature)`